build-scripts/rcm-guest/push-to-mirrors.sh

```diff
@@ -83,12 +83,15 @@ chmod 775 "${PUDDLEDIR}/" || true
 # dereference the building symlink to the actual directory basename: e.g. "2017-06-09.4"
 LASTDIR=$(readlink --verbose "${PUDDLEDIR}/building")
 
+# use time of last data modification of building dir as suffix
+MODIFIED_TIMESTAMP=$(stat -c %Y ${LASTDIR})
+
 # Create a symlink on rcm-guest which includes the OCP version. This
 # helps find puddles on rcm-guest for particular builds. Note that
 # we can't simply rename the directory, because the directory contains
 # puddle.repo contains a URL referring to the puddle directory name
 # that was created by the puddle command.
-VERSIONED_DIR="v${FULL_VERSION}_${LASTDIR}"  # e.g. v3.7.0-0.173.0_2017-06-09.4
+VERSIONED_DIR="v${FULL_VERSION}_${MODIFIED_TIMESTAMP}"  # e.g. v3.7.0-0.173.0_2017-06-09.4
```

Comment:
I didn't see the point of this PR. Currently `plashet_dir/building` is always pointing to `plashet_dir/stream/building`.

Reply:
This PR only solves the issue that `VERSIONED_DIR` now contains a `/` (e.g. `v3.11.457-1_stream/building`). 

Historically,  `${PUDDLEDIR}/building` was a symlink to specific plashet dir but now it is another symlink to `${PUDDLEDIR}/stream/building`. Pointing `VERSIONED_DIR` to `${PUDDLEDIR}/stream/building` is now meaningless because `${PUDDLEDIR}/stream/building` is changing every time we have a new plashet.

I would recommend to change `LASTDIR` to `$(readlink --verbose "${PUDDLEDIR}/stream/building")`.